src/index.ts

```diff
@@ -408,11 +493,14 @@ export type DeclarativeMutation = {
   attribute: string;
   action: 'append' | 'set' | 'remove';
   value?: string;
+  parentSelector?: string;
+  insertBeforeSelector?: string;
 };
 
 export default {
   html,
   classes,
   attribute,
+  move,
```

Comment:
I'm thinking we get rid of all uses of the word `move` and use `position` instead to keep naming consistent.  So `positionMutationRunner`, `getElementPositionRecord`, etc.

Reply:
Updated